src/sass/components/_brands.scss

```diff
@@ -0,0 +1,92 @@
+.section-brands {
+  margin-bottom: 50px;
+  border: 1px solid $brans-border-color;
+  .flickity-viewport {
+    width: 100%;
+    height: 150px;
+  }
+  .flickity-prev-next-button.previous {
+    left: 0;
+  }
+  .flickity-prev-next-button.next {
+    right: 0;
+  }
+
+  .flickity-page-dots {
+    z-index: 3;
+    width: 100%;
+    height: 20px;
+    padding: 0 10px;
+    .dot {
+      width: 13px;
+      height: 13px;
+      margin: 0 6px;
+      opacity: 1;
+      background: #e2e2e2;
+      &:hover,
+      &.is-selected {
+        background: $primary;
+      }
+    }
+  }
+
+  .carousel-cell {
+    width: 100%;
+    height: 100%;
+    padding: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .brands-image {
+    filter: grayscale(100%);
+    width: 100%;
+    object-fit: cover;
+    &:hover {
+      filter: grayscale(0);
+    }
+  }
+
+  .flickity-prev-next-button {
+    height: 68px;
+    width: 15px;
+    padding: 0;
+    border: none;
+    border-radius: 0;
+    margin-left: 0;
+    background: $footer-claim-bg;
+    color: white;
+    font-size: 1em;
+    &:hover {
+      background: $primary;
+    }
+
+    i {
+      justify-content: center;
+      align-content: center;
+    }
+  }
+}
+
+@media (max-width: 576px) {
+  .section-brands {
+    .flickity-viewport {
+      height: 400px;
+    }
+    .brands-image {
+      object-fit: contain;
+    }
+  }
+}
+
+@media (max-width: 768px) {
+  .section-brands {
+    .flickity-viewport {
+      height: 220px;
+    }
+    .brands-image {
+      object-fit: contain;
+    }
+  }
+}
```

Comment:
To dobrze działa w praktyce? Nie powinno być ustawionych minimalnych wartości dla width?
Bo teraz chyba i `height: 220px i height: 400px` pasuje do kryteriów `max-width: 768px`